pkgs/applications/altcoins/spruned/default.nix

```diff
@@ -0,0 +1,47 @@
+{ stdenv, pythonPackages }:
```

Comment:
Take the `python3Packages` argument instead. Or the `python3` argument (then `python3.pkgs` to get the packages).

Then you won't have to pass it in `altcoins/default.nix`.